### DISCUSSION TITLE
Call an organ for opening in favorite

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1824] [Help]
Hello all,

Is it possible with a Midi command (for example configuring a button on my console) to call an organ ?

For example I start an organ (Friesach) and without screen, opening another organ. 
For that I want to configure a button or piston on my console to ask the organ I want. 
Is that possible ?

**Comment by oleg68** [comment]
File->Settings->Organs->MIDI

**Reply by Jean-Gabriel1982** [reply]
Many thanks !!
